web/src/app/util/luxon-helpers.js

```diff
@@ -0,0 +1,33 @@
+import { DateTime } from 'luxon'
+
+// getLuxonStartOfWeek
+// Given a JS Date object, this yields a DateTime object set to the previous Sunday at 12am.
+// If no date is provided, default to now
+// This is excluded from Luxon core since certain countries start the week on Saturday or Sunday
+// Conventionally, weeks start on a Monday
+export function getLuxonStartOfWeek(JSDate = new Date()) {
```

Comment:
I think it would be clearer to just call this `getStartOfWeek` or `startOfWeek` and operate on `DateTime` directly, doing the to/from jsDate stuff only in the specific places we need to (i.e. calendar) instead of in the helper.